qiskit/compiler/transpiler.py

```diff
@@ -341,6 +341,15 @@ def callback_func(**kwargs):
         target = copy.deepcopy(target)
         target.update_from_instruction_schedule_map(inst_map)
 
+    if translation_method == "unroller":
+        warnings.warn(
+            "The 'unroller' translation_method plugin has been deprecated and will be removed "
+            "in a future release. Instead you should use the default 'translator' method or "
+            "another plugin",
```

Comment:
I think it would be helpful to include the qiskit version in the warning:

```suggestion
            "The 'unroller' translation_method plugin is deprecated as of qiskit-0.45.0 and "
            "will be removed in a future release. Instead you should use the default "
            "'translator' method or another plugin.",
```